tekton/pipelines/binary-container.yaml

```diff
@@ -0,0 +1,106 @@
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: binary-container-0-1  # dot is not allowed in the name
+spec:
+  params:
+    - name: OSBS_IMAGE
+      description: The location of the OSBS builder image (FQDN pullspec)
+      type: string
+    - name: USER_PARAMS
+      type: string
+      description: User parameters in JSON format
+
+  workspaces:
+    - name: ws-build-dir
+    - name: ws-context-dir
+    - name: ws-registries-secret
+    - name: ws-koji-secret
+    - name: ws-worker-tokens-secrets
```

Comment:
What is `worker-tokens-secrets`?

Reply:
we should remove it, it was for ocp tokens for worker

Reply:
+1, we might have need some secret for the remote build hosts but I wouldn't worry about those yet

Reply:
piece of outdated design. Removed it.